cli/functionary/package.py

```diff
@@ -84,3 +86,43 @@ def publish(ctx, path):
         response = post("publish", files={"package_contents": upload_file})
         id = response["id"]
         click.echo(f"Package upload complete\nBuild id: {id}")
+
+
+@package_cmd.command()
+@click.pass_context
+@click.option("--id")
+def buildstatus(ctx, id):
+    """
+    View status for all builds, or build with specific id
+    """
+    if id:
+        results = get(f"builds/{id}")
+        _format_results([results], title=f"Build: {id}")
+    else:
+        results = get("builds").get("results")
+        _format_results(results, title="Build Status")
+
+
+def _format_results(results, title=""):
+    """
+    Helper function to organize table results using Rich
+
+    Args:
+        results: Results to format as a List
+        title: Optional table title as a String
+
+    Returns:
+        None
+    """
+    table = Table(title=title, width=170)
+    console = Console()
+    first_row = True
+    for item in results:
+        list = []
+        for key in item:
+            if first_row:
+                table.add_column(key.capitalize())
+            list.append(str(item[key]))
```

Comment:
As written, this ends up stringifying `None` values, making them show up as "None" rather than a blank cell, which is not ideal.

You can avoid that by doing this:

```python
list.append(str(item[key]) if item[key] else None)
```

Doing that leads to you resolving `item[key]` multiple times, which we should also avoid.  You can actually get the key and value at the same time at the top of your for loop by doing:

```python
for key, value in item.items():
```

Then when you get down here, you have the more sensible looking:

```python
list.append(str(value) if value else None)
```